index.html

```diff
@@ -41,5 +41,146 @@ <h1 class="primary-text">I'm Mariene Glad to see you!</h1>
+            <div class="details-container">
+                <ul>
+                    <li class="main-item">canopy</li>
+                    <li class="sub-item">Back End Dev</li>
+                    <li class="sub-item">2015</li>
+                </ul>
+                <p class="card-text">
+                    A daily selection of privately
+                    personalized reads: no accounts 
+                    or sign-ups required.
+                </p>
+                <ul class="tech-used">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javascript</li>
+                </ul>
+                <a class="see-project" href="#">See Project</a>
+            </div>
+        </div>
+        <div class="card">
+            <img class="card-img" src="./img/pic4.png" alt="first-project">
+            <h2 class="main-title">Multi-Post Stories</h2>
+            <div class="details-container">
+                <ul>
+                    <li class="main-item">canopy</li>
+                    <li class="sub-item">Back End Dev</li>
+                    <li class="sub-item">2015</li>
+                </ul>
+                <p class="card-text">
+                    A daily selection of privately
+                    personalized reads: no accounts 
+                    or sign-ups required.
+                </p>
+                <ul class="tech-used">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javascript</li>
+                </ul>
+                <a class="see-project" href="#">See Project</a>
+            </div>
+        </div>
+        
+    </section>
+    <section class="about-myself">
+        <div class="text-holder">
+            <h1 class="primary-text-about">About Myself</h1>
+            <p class="details">
+                I'm a software developer! I can help you
+                build a product, feature or website Look
+                through some of my work and 
+                Experience! If you like what you see and
+                have a project you need coded, don't 
+                hestiate to contact me.
+            </p>
+            <a href="#" class="connect">Let's connect</a>
+        </div>
+        <div class="fa-container-about">
+            
+              <i class="fa-brands fa-twitter fa"></i>
+              <i class="fa-brands fa-linkedin-in fa"></i>
+              <i class="fa-brands fa-medium fa" ></i>
+              <i class="fa-brands fa-github fa"></i>
+              <i class="fa-solid fa-hand-peace fa"></i>
+        </div>
+        <a class="see-project-about" href="#">Get My Resume!</a>
+        <div class="details-2">
+            <ul>
+                <li>
+                    <div class="details-2-content">
+                       <p class="details-2-title">Languages</p>
+                       <i class="fa-solid fa-angle-down"></i>
+                    </div>
+                    <hr>
+                </li>
```

Comment:
- [ ] You should add the languages ( "JavasScript", "HTML" and "CSS") as in the design, and align this section correctly, this will make your design more professional

|Your design|Figma design|
|-|-|
|![Screenshot_20220701_045248](https://user-images.githubusercontent.com/25227022/176820568-bfe9b0ae-919d-4c0b-aeca-33e43346e54e.png)|
![Screenshot_20220701_045903](https://user-images.githubusercontent.com/25227022/176820764-4c62b5a6-f358-42c5-b3bd-90d78489fd11.png)|